### PR TITLE
SqsConnector

### LINKED ISSUE #106: AWS SQS - new connector
Using  https://github.com/softwaremill/elasticmq for local testing.

### PR DESCRIPTION
**Pull request by xplosunn**
Closes #106

### CODE REVIEW

File: project/SqsDependencies.scala [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import sbt._
```
Comment (adrianfilip) [inline project/SqsDependencies.scala:1]: We should use the latest version of aws sdk. https://github.com/zio/zio-connect/blob/master/project/AWSLambdaDependencies.scala

File: connectors/sqs-connector/src/main/scala/zio/connect/sqs/SqsConnector.scala [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package zio.connect.sqs
+
+import zio.connect.sqs.SqsConnector.{ReceiveMessage, SendMessage, SendMessageBatch, SqsException}
+import zio.prelude.Newtype
+import zio.stream.{ZSink, ZStream}
+import zio.{Trace, ZIO}
+
+trait SqsConnector {
```
Comment (adrianfilip) [inline connectors/sqs-connector/src/main/scala/zio/connect/sqs/SqsConnector.scala:8]: Should be ordered alphabetically

File: project/SqsDependencies.scala (hunk 1 above)
Comment (adrianfilip) [reply project/SqsDependencies.scala:1]: I we should use zio-aws and build this connector on top of it. https://github.com/zio/zio-aws

**Comment by adrianfilip** [comment]
Hi @xplosunn I think we can merge this after you get the tests to pass.

File: project/SqsDependencies.scala (hunk 1 above)
Comment (xplosunn) [reply project/SqsDependencies.scala:1]: It's currently using the same versions as the S3 package. Maybe they should both be upgraded together? Or would you rather I update this one?

**Comment by xplosunn** [comment]
@adrianfilip sorry for the delay. Just updated the PR.

**Pull request closed without merging**